app/src/main/res/layout/activity_call.xml

```diff
@@ -248,7 +249,7 @@
         app:layout_constraintStart_toEndOf="@+id/linearLayout_transfer"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_bias="0.74"
-        tools:ignore="MissingConstraints">
+        tools:ignore="MissingConstraints" >
```

Comment:
You want to add in the constraints that it is complaining about rather than ignoring them.